sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/windowing/CalendarWindows.java

```diff
@@ -229,6 +237,13 @@ public boolean isCompatible(WindowFn<?, ?> other) {
           && timeZone == that.timeZone;
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("numMonths", number)
+          .add("startDate", new DateTime(startDate, timeZone).toInstant());
```

Comment:
This could benefit from the "omit default" logic (I believe we use the epoch as the default).